src/MultivariateStats.jl

```diff
@@ -563,8 +595,8 @@ $BayesianSubspaceLDA_DESCR
                     to be used by `predict` and `transform` methods, automatically set if 0 is given (default).
 * `priors=nothing`: For use in prediction with Baye's rule. If `priors = nothing` then `priors` are estimated
                     from the class proportions in the training data. Otherwise it requires a vector containing 
-                    class probabilities specified by order of class levels ommitting class levels which are not
-                    present in training data.
+                    class probabilities specified in order following the ordering of levels found in the common pool
+                    of the target vector.
 
```

Comment:
Minor suggestion: You could be a little more explicit and say something like "... using the order given by `levels(y)`, where `y` is the target vector. "

Reply:
@ablaom i think this is okay.
```
* `priors=nothing`: For use in prediction with Baye's rule. If `priors = nothing` then `priors` are estimated
                    from the class proportions in the training data. Otherwise it requires a `Vector` containing 
                    class probabilities with probabilities specified using the order given by `levels(y)` where y is the
                    target vector.
```